src/shared/utilities/requestUtils.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs'
+// TODO: Move off of deprecated `request` to `got` or similar modern library.
+import * as request from 'request'
+
+/**
+ * Makes a `get` request to a URL without params and returns the response asynchronously.
+ * Can optionally pipe the call's data to a location.
+ * Swallows error to hide the targeted URL. Recommended by security.
+ * @param url URL to send request to
+ * @param pipeLocation Filepath to pipe output to.
+ */
+export async function getResponseFromGetRequest(url: string, pipeLocation?: string): Promise<request.Response> {
```

Comment:
could live in `HttpResourceFetcher`. Either `HttpResourceFetcher` or this requestUtils is redundant. Better to keep related code together than spread it out in many tiny files.

this also looks very similar to the purpose of `HttpResourceFetcher`.

Reply:
This one is specifically to accommodate the pipe. I'll play with it a little more to see if I can wire that up there.

Reply:
it could still live in `HttpResourceFetcher` (as a module function or a static class function) or whatever module we want to keep HTTP stuff in.

Reply:
Moved